internal/cmd/api_common.go

```diff
@@ -56,11 +58,13 @@ func getTrace() bool       { return trace || traceAll }
 func getTraceAll() bool    { return traceAll }
 func getEnableChaos() bool { return enableChaos }
 func getAPIToken() string  { return apiToken }
+func getTokenDir() string  { return tokenDir }
 
 func setAPIFlags(flagSet *pflag.FlagSet) {
 	flagSet.StringVar(&authBaseURL, authURLF, defaultAuthURL, "Backend URL to authorize you")
 	flagSet.StringVar(&apiBaseURL, apiURLF, "https://api.stateful.com", "Backend URL with API")
 	flagSet.StringVar(&apiToken, apiTokenF, "", "API token")
+	flagSet.StringVar(&tokenDir, tokenDirF, getDefaultConfigHome(), "Location dir where token will be save")
```

Comment:
I think that "config directory" makes a lot more sense than "token directory"

IMO we should change the language around throughout the codebase so it's e.g. `--config-dir` rather than `--token-dir`. Just an opinion though ¯\\\_(ツ)_/¯

Reply:
Yes I like that one better :)